tutorials/source-modeling/plot_source_alignment.py

```diff
@@ -209,50 +216,48 @@ def plot_dig_alignment(points, coord_system='ras'):
-#
-# The plot below shows the head space coordinates transformed to meg
-# coordinates. It correctly aligns to the head, but still has to be transformed
-# to the mri coordinate space, which is in voxels. In mri (voxel) space,
-# ``(0, 0, 0)`` is the left-, posterior-most coordinate on the inferior-most
-# slice compared to ``(0, 0, 0)`` being the center of the head in both head
-# and meg coordinate spaces.
+# Next, we'll apply the precomputed ``trans`` to convert the digitized points
+# from "head" to "meg" coordinate frame. It looks pretty good, but the
+# coordinate frames aren't actually perfectly aligned yet. This underscores
+# why interactive use of `~mne.viz.plot_alignment` is such an important step:
+# static plots can sometimes *look okay* even if they're not actually aligned
+# properly.
 
 meg_space = mne.transforms.apply_trans(trans, head_space, move=True)
 plot_dig_alignment(meg_space)
 
 ###############################################################################
-# Apply a second transform to get to mri space and plot again
-# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+# Finally, we'll apply a second transformation to the already-transformed
+# digitized points. That transform comes from the T1 header, and converts from
+# native MRI voxels (called the "mri_voxel" coordinate frame in MNE-Python) to
+# the "mri" coordinate frame (MRI Surface RAS). Unlike MRI Surface RAS,
+# "mri_voxel" has its origin in the corner of the volume (the left-most,
+# posterior-most coordinate on the inferior-most MRI slice) instead of at the
+# center of the volume. "mri_voxel" is also **not** an RAS coordinate system:
+# rather, its XYZ directions are based on the acquisition order of the T1 image
+# slices, so we'll need to do some extra steps here.
 #
-# This plot finally shows the coordinates in the mri space, which looks the
-# same as the alignment meg coordinate space, but as can be seen in the
-# helper function we actually have to do a transformation back from mri to
-# meg space. This is because the freesurfer surfaces (the reconstructed head
-# surface that is plotted) are centered in the center of the head and in RAS
-# coordinates compared to the mri coordinate space which is centered in the
-# bottom corner and the order of the points is not RAS but based on the T1
-# image slices.
+# .. note::
+#     Normally, MNE-Python converts ``mri_voxel → mri`` coordinate frame
+#     automatically before displaying skin/skull/brain surfaces, so the extra
+#     steps aren't needed for most analysis tasks.
 
 vox2ras_tkr = t1_mgh.header.get_vox2ras_tkr()
 ras2vox_tkr = linalg.inv(vox2ras_tkr)
-mri_space = mne.transforms.apply_trans(
-    ras2vox_tkr, meg_space * 1e3)  # units must be in mm, scaled from m
-plot_dig_alignment(mri_space, coord_system='mri')
+
+vox_space = mne.transforms.apply_trans(ras2vox_tkr, meg_space * 1e3)  # m → mm
+mri_space = ((vox_space - 128) * [1, -1, 1])[:, [0, 2, 1]] * 1e-3
```

Comment:
@alexrockhill I think the main source of confusion for me was that in your function you were referring to two coordinate systems as "ras" and "mri", which in MNE-Python (internally) those are called "mri" and "mri_voxel", respectively.  That clears up the main confusion on my end, but there's still something I don't understand here:  why are you applying a `ras2vox` transform to something that is in *meg* coordinate frame?  Aren't we missing a step of transforming MEG frame to MRI Surface RAS first?

Reply:
If I'm not mistaken, the `trans` defines the transform from head to MRI surface RAS, combining the transformation from head to meg and meg to mri_ras so I think this should be changed to reflect that, but maybe let's check with @larsoner before going with that answer

Reply:
In that case we shouldn't call that variable `meg_space`, and some of the prose needs to change.

Reply:
Yeah that's how it was before and then I changed it to meg_space to match from mri_space but I don't think that was right

Reply:
Ok I changed it the way I think it should be